### ISSUE TITLE
Conditionally revealing content with checkboxes not announcing collapsed/expanded state on macOS with VoiceOver

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk_elements#575]
When using the [conditionally revealing content component](http://govuk-elements.herokuapp.com/form-elements/#form-toggle-content) VoiceOver on macOS in Safari doesn't announce that it is collapsed or expanded, despite the use of aria-expanded and others. All other screen readers are fine with it. Weirdly, that only happens when using checkboxes, not radio buttons!

## How would you expect it to work?

When on the trigger ("Citizen of a different country"), VoiceOver should say "collapsed" (like it does when using the same component with radio buttons). When triggering it, it should say "expanded".

## How does it work currently?

It doesn't say whether it's collapsed or expanded.

## Feel free to suggest a fix...

I tried a few things (removing aria-hidden, removing aria-controls and moving aria-expanded) but nothing I tried fixed it. It's probably a bug in VoiceOver and/or Safari, but I wasn't able to make a reduced test case, so haven't reported it.

I was wondering if this bug has anything to do with #455 (or alphagov/govuk_frontend_toolkit#419), but the invalid ARIA attribute that issue is talking about is only invalid for radio buttons, not checkboxes. And #573 is thematically related but otherwise probably not at all.

It's very unlikely that this bug will create a barrier as tabbing or down-arrowing or moving to the next form field would always go to the just revealed form field. As such, fixing this has a low priority.

**Comment by nickcolley** [comment]
Following the launch of the GOV.UK Design System, GOV.UK Elements will now only get major bug fixes and security patches.

This is definitely still an issue, and we will investigate this and see if we can resolve it in GOV.UK Frontend: https://github.com/alphagov/govuk-frontend/issues/989

Thanks so much for these great issues!

**Comment by joshueoconnor** [comment]
FYI I tested this component on Gov Elements and the state info (expanded/collapsed) was announced in Safari 20 with VoiceOver on macOS.
